Project 1 Day 2/templates/admin.html

```diff
@@ -0,0 +1,44 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Required meta tags -->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+
+    <!-- Bootstrap CSS -->
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css" integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+    <title>Book - Read,Rate,Review</title>
+    <link rel = "stylesheet" href = "{{ url_for('static', filename='custom.css') }}">
+    <link rel="icon" href="https://img.freepik.com/free-vector/hand-drawn-world-book-day-concept_23-2148469171.jpg?size=338&ext=jpg" type="image/jpg" sizes="16x16">
+  </head>
```

Comment:
All of this could be part of a generic template as it may be repeating itself many web pages.